td-agent/debian/rules

```diff
@@ -13,6 +13,7 @@ override_dh_auto_install:
 	rake build:deb_config TD_AGENT_STAGING_PATH="$(CURDIR)/debian/tmp"
 	rake build:all        TD_AGENT_STAGING_PATH="$(CURDIR)/debian/tmp"
 	dh_installman
+	find $(CURDIR) -name '.git*' -print0 | xargs -0 rm -fr
```

Comment:
I want to remove these files from RPM too, so it should be done in Rakefile.

Reply:
Thanks, I've fixed it.